### PR TITLE
Add pycodestyle check in github action and Makefile

### PR DESCRIPTION
**Pull request by eddyxu**
`make lint` now support both pycodestyle and black

### CODE REVIEW

**Comment by eddyxu** [comment]
`flake8` seems to have much more warnings, we can do it later.

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+exclude =
+    .git,
+    __pycache__,
+    venv,
+    dist,
+    build
+max-line-length = 99
+
+[pycodestyle]
+max-line-length = 99
```
Comment (changhiskhan) [inline setup.cfg:11]: 79 or 99?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+max-doc-length= 200
```
Comment (changhiskhan) [inline setup.cfg:12]: is this PEP8 compliant?

Comment (eddyxu) [reply setup.cfg:12]: How to handle long URLs in doc? 

We can strictly follow with PEP8 and disable lint on such lines manually. 
What do you think?

File: setup.cfg (hunk 1 above)
Comment (eddyxu) [reply setup.cfg:11]: Deliberately change this to `99` from the default value (79).

I am fine to stick on `79`

File: setup.cfg (hunk 2 above)
Comment (changhiskhan) [reply setup.cfg:12]: using `noqa`? I think that's fine

**Review by changhiskhan — APPROVED** [review]
bombs away

**Pull request merged**